PermissionsScraper/PermissionsScraper/Triggers/DescriptionsScraper.cs

```diff
@@ -67,7 +66,7 @@ public static void Run([TimerTrigger("%ScheduleTriggerTime%")] TimerInfo myTimer
                     }
                 }
 
-                if (!_servicePrincipalPermissions.Any())
+                if (_servicePrincipalPermissions.Count == 0)
```

Comment:
why remove any?

Reply:
I was getting this error when I bumped up the project to use net5.0 
`'System.Linq, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The system cannot find the file specified.`

Reply:
`C:\Users\runneradmin\.nuget\packages\microsoft.net.sdk.functions\3.0.3\build\Microsoft.NET.Sdk.Functions.Build.targets(41,5): error : It was not possible to find any compatible framework version [D:\a\msgraph-permissions-scraper\msgraph-permissions-scraper\PermissionsScraper\PermissionsScraper\PermissionsScraper.csproj]` leads me to think the azure functions SDK on the agents might be outdated. Maybe adding a reference to 3.0.13 would fix this issue.
https://www.nuget.org/packages/Microsoft.NET.Sdk.Functions/3.0.13

Reply:
additional resources https://docs.microsoft.com/en-us/azure/azure-functions/dotnet-isolated-process-guide
https://techcommunity.microsoft.com/t5/apps-on-azure/net-on-azure-functions-roadmap/ba-p/2197916

Reply:
https://github.com/microsoftgraph/msgraph-permissions-scraper/pull/18/files#diff-f13d431b675a71faf2e36d49c905263a140f64a4ae96f96d6c183aad3c6059de

Reply:
Tried bumping up to this version: https://www.nuget.org/packages/Microsoft.NET.Sdk.Functions/3.0.13 but still error-ing out. After reading this article you shared: https://techcommunity.microsoft.com/t5/apps-on-azure/net-on-azure-functions-roadmap/ba-p/2197916, I'm of the opinion we can just stick to `netcoreapp3.1` in the interim until we are sure of full support for `net5.0` for the .NET isolated process in Azure Functions.